### PR TITLE
fix: resolve dependency and environment management issues 

### PR DESCRIPTION
**Pull request by jsstevenson**
Take 2

Fixes a couple of environment/dependency issues that cropped up recently

### CODE REVIEW

**Comment by korikuzma** [comment]
@jsstevenson did you want to try and resolve the GH Actions before I review?

**Comment by jsstevenson** [comment]
@korikuzma I'm not touching the tests until https://github.com/cancervariants/therapy-normalization/tree/mock_data is merged. Up to you if you want to wait til then, we don't per se *need* to merge this imminently

**Comment by korikuzma** [comment]
> @korikuzma I'm not touching the tests until https://github.com/cancervariants/therapy-normalization/tree/mock_data is merged, they make me crazy and bring me stress. Up to you if you want to wait til then, we don't per se _need_ to merge this imminently

@jsstevenson Ah okay, that's fine.

**Review by korikuzma — DISMISSED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -38,7 +38,7 @@ dev =
     ipykernel
     pre-commit
     tox
-    flake8
```
Comment (korikuzma) [inline setup.cfg:41]: Do we still need tox?

**Review by korikuzma — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (jsstevenson) [reply setup.cfg:41]: the prophecy is in tatters :(

**Pull request merged**